### PR TITLE
Allow for multiple dashboard urls to be provided and rendered for a c…

### PR DESCRIPTION
**Pull request by jsidney**
The current Datadog plugin only allows for a single dashboard to be rendered through the `datadoghq.com/dashboard-url:` annotation.  There are going to be use-cases where a certain component will have multiple dashboards that need to be displayed through the plugin. This change allows for a comma-separated list of dashboard URLs to be passed through this annotation and rendered on the component page.
#### :heavy_check_mark: Checklist

- [ ] Added tests for new functionality and regression tests for bug fixes
- [x] Added changeset (run `yarn changeset` in the root)
- [x] Screenshots of before and after attached (for UI changes)
- [x] Added or updated documentation (if applicable)

### CODE REVIEW

**Comment by jsidney** [comment]
## Current display: 
datadog_current_screenshot

## Proposed change:
datadog_proposed_screenshot

File: plugins/frontend/backstage-plugin-datadog/src/components/DatadogDashboardPage.tsx [hunk 1]
```diff
@@ -17,33 +17,41 @@
...
-            width: '100%',
-            resize: 'both',
-            overflow: 'auto',
-          }}
-        />
-      </Resizable>
-    </InfoCard>
+    <Grid container spacing={3}>
+      {allDashboardUrls.map((value, index) => (
+        <Grid item md={12}>
```
Comment (Xantier) [inline plugins/frontend/backstage-plugin-datadog/src/components/DatadogDashboardPage.tsx:31]: Could you add a key to this component?

Comment (jsidney) [reply plugins/frontend/backstage-plugin-datadog/src/components/DatadogDashboardPage.tsx:31]: @Xantier  - i assume by key, you mean like and index? (eg: name of the dashboard or something?)

**Comment by jsidney** [comment]
@Xantier  - i want to add to the mock so that the tests for this plugin are also up to date.... my question is that I cant see how the `plugin.test.tsx` calls the instantiates the mock. so before I add to the mock, I want to just make sure I do it correctly. can you please point me in the right direction so that I can add the tests correctly?

File: plugins/frontend/backstage-plugin-datadog/src/components/DatadogDashboardPage.tsx (hunk 1 above)
Comment (Xantier) [reply plugins/frontend/backstage-plugin-datadog/src/components/DatadogDashboardPage.tsx:31]: More info: https://react.dev/learn/rendering-lists. the name or maybe the annotation itself should work here.

**Comment by Xantier** [comment]
@jsidney, it looks like Datadog plugin doesn't have tests written for it yet. You can probably take a look at an example from argocd plugin for example and see how they handle the mocks in that package.

File: plugins/frontend/backstage-plugin-datadog/src/components/DatadogDashboardPage.tsx (hunk 1 above)
Comment (jsidney) [reply plugins/frontend/backstage-plugin-datadog/src/components/DatadogDashboardPage.tsx:31]: i have added the key 👍

**Review by Xantier — APPROVED** [review]

File: plugins/frontend/backstage-plugin-datadog/src/components/DatadogDashboardPage.tsx [hunk 2]
```diff
@@ -17,33 +17,41 @@
...
+          <InfoCard title={`Datadog dashboard ${index}`} variant="gridItem">
+            <Resizable
+              defaultSize={{
+                width: 100,
```
Comment (stasOR41k) [inline plugins/frontend/backstage-plugin-datadog/src/components/DatadogDashboardPage.tsx:35]: should it be reverted to the '100%'?
I have an issue with the default size in the [2.1.15](https://www.npmjs.com/package/@roadiehq/backstage-plugin-datadog/v/2.1.15)

Comment (jsidney) [reply plugins/frontend/backstage-plugin-datadog/src/components/DatadogDashboardPage.tsx:35]: More than happy to make that change @stasOR41k  - I think it must have been something I had changed without noticing while playing around with the look and feel

**Pull request merged**